internal/cas/fetch.go

```diff
@@ -69,3 +69,31 @@ func FetchManifest(ctx context.Context, target oras.Target, reference string, p
 	defer rc.Close()
 	return content.ReadAll(rc, desc)
 }
+
+// FetchBlob fetches the blob content of reference from blob store.
+func FetchBlob(ctx context.Context, blob oras.Target, reference string) ([]byte, error) {
```

Comment:
Let's use [oras.Fetch](https://github.com/oras-project/oras-go/blob/main/content.go#L119) and [oras.FetchBytes](https://github.com/oras-project/oras-go/blob/main/content.go#L169)! 😀